crates/sui-indexer-alt/src/handlers/obj_info.rs

```diff
@@ -18,7 +19,12 @@ use sui_types::{
     object::{Object, Owner},
 };
 
-pub(crate) struct ObjInfo;
+use crate::consistent_pruning::{PruningInfo, PruningLookupTable};
+
+#[derive(Default)]
+pub(crate) struct ObjInfo {
+    pruning_lookup_table: Arc<PruningLookupTable>,
+}
```

Comment:
I think I might be missing something, but I don't see the consistent_pruning changes in this PR, or on main?

Reply:
Ah forgot to include the file